src/node.js

```diff
@@ -984,7 +988,8 @@
 
   NativeModule.prototype.compile = function() {
     var source = NativeModule.getSource(this.id);
-    source = NativeModule.wrap(source);
+    var cached_data = NativeModule.getSourceCachedData(this.id);
+    source = NativeModule.wrap(source, cached_data);
```

Comment:
Looking at the definition of `NativeModule.wrap` 12 lines up - how is `cached_data` being used?